src/openforms/submissions/api/viewsets.py

```diff
@@ -110,60 +104,64 @@ def _complete(self, request, *args, **kwargs):
 
         Submission completion requires that all required steps are completed.
 
-        Once a submission is completed, it's removed from the session. This means it's
-        no longer possible to change or read the submission data (including individual
-        steps).
+        Note that the processing of the submission runs in the background, and you
+        should periodically check the submission status endpoint to check the state.
+        Background processing makes sure that:
+
+        * potential appointments are registered
+        * a report PDF is generated
+        * the submission is persisted to the configured backend
+        * payment is initiated if relevant
+
+        Once a submission is done processing, it's scheduled to be removed from the
+        session. This means it's no longer possible to change or read the submission
+        data (including individual steps).
 
-        The submission is persisted to the configured backend.
+        TODO: give some leeway? add endpoint to trigger cleanup immediately?
```

Comment:
Sounds like a TODO for when we implement "Tijdelijk opslaan", as described here: https://github.com/open-formulieren/open-forms/issues/227

Reply:
bwoah, leaning more towards the timestamped HMAC'd token approach for that, similar to the download report URL